docs/fsharp/tour.md

```diff
@@ -21,7 +21,7 @@ There are two primary concepts in F#: functions and types.  This tour will empha
 ## How to Run the Code Samples
 
 >[!NOTE]
-Eventually, this documentation site will have the ability to run these code samples directly in your browser.  If your browser supports Silverlight, you can use [TryFsharp](http://www.tryfsharp.org/Create) for the time being.
+Eventually, this documentation site will have the ability to run these code samples directly in your browser. You can choose two way: Try [F# for Azure Notebooks](https://notebooks.azure.com/Microsoft/libraries/fsharp/html/FSharp%20for%20Azure%20Notebooks.ipynb) on Microsoft Azure Notbooks (Currently preview, free to use.) Or, if your browser supports Silverlight, you can use [TryFsharp](http://www.tryfsharp.org/Create) for the time being.
```

Comment:
Thanks @kekyo. Would you mind removing this line from the start of the note? ...

> Eventually, this documentation site will have the ability to run these code samples directly in your browser. 

We removed this line from another topic because there is no immediate plan to implement a code editor in these docs. We should remove it here, too.